src/pages/storage/StorageVolumesFilter.tsx

```diff
@@ -71,32 +62,40 @@ const StorageVolumesFilter: FC<Props> = ({ volumes, setFilters }) => {
   ];
 
   const onSearchDataChange = (searchData: SearchAndFilterChip[]) => {
-    const getChipValue = (lead: string) =>
+    const searchTokensByLead = (lead: string) =>
       searchData.filter((chip) => chip.lead === lead).map((chip) => chip.value);
 
-    setFilters({
-      queries: searchData
-        .filter((chip) => chip.quoteValue)
-        .map((chip) => chip.value.toLowerCase()),
-      pools: getChipValue(POOL),
-      volumeTypes: getChipValue(VOLUME_TYPE),
-      contentTypes: getChipValue(CONTENT_TYPE),
+    const newParams = new URLSearchParams();
+    searchTokensByLead("quote").forEach((query) => {
+      newParams.append("q", query);
+    });
```

Comment:
This is broken - see the discussion [here](https://github.com/canonical/lxd-ui/pull/536#discussion_r1394248647). Please apply the same fix as in the other PR.

Reply:
Ah right, I extracted the generic parts to a util, so we can reuse it easily now.